src/kata5.reachDestination.js

```diff
@@ -1,3 +1,5 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+ const timeToArrival = Math.ceil((distance/speed)*2)/2;
```

Comment:
Awesome. Only slight suggestion here would be that timeToArrival is actually a string and I would assume this was a timestamp in a real world scenario so would change it to something like `timeToArrivalString`